doc/sphinxman/source/add_tests.rst

```diff
@@ -0,0 +1,79 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2017 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. _`sec:add_tests`:
+
+Adding Test Cases
+=========
```

Comment:
sphinx, not me, demands that the number of `=` underlining the section title equal or exceed the length of the section title.

Reply:
Weird, neither sphinxmini nor sphinxman gave me an error when I made the documentation. I shouldn't be surprised that sphinx is so cryptic.

Reply:
it may be just a warning. once upon a time I had a clean sphinx build and so could notice these things. if you misalign a table cell delimiter character on their fancy tables (e.g., the energy docstring one in driver.py), it will error.